packages/sources/paypal/src/endpoint/getPayout.ts

```diff
@@ -41,7 +41,12 @@ export const execute: ExecuteWithConfig<Config> = async (request, config) => {
   try {
     response = await config.api.client.execute(paypal_req)
   } catch (e) {
-    throw Requester.errored(jobRunID, e)
+    e.message = JSON.stringify({ statusCode: e.statusCode, ...JSON.parse(e.message) })
+    throw new AdapterError({
+      jobRunID,
+      message: e.message,
+      statusCode: 500,
+    })
```

Comment:
- It would be nice to use `Requester.success` when the call succeeds, so we make sure everything stays uniform.
- About the error, why not forward the same error status code from our adapter? At the end, the paypal client is just making an HTTP request as well
```suggestion
try {
    const response = await config.api.client.execute(paypal_req)
    return Requester.success(jobRunID, { ...response, status: response.statusCode })
  } catch (e) {
    throw Requester.errored(jobRunID, e, e.statusCode)
  }
```

Reply:
Good points! I will make some changes to use the standard `Requester.success` and `Requester.errored`

My original intention was to standardize the error code that the adapter returned since i saw different errors ranging from 400, 404, and 429 which made it hard to test. But I think I might need to rethink how the test cases are written.

Also for the `success` return, the response structure doesn't match the expected input structure so that's why it's custom. But I think you're right in that it would be useful to standardize the output.